Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -221,13 +221,26 @@ public class HTTPClient {
         }
     }
 
+    /// Specifies redirect processing settings.
+    public typealias RedirectConfiguration = Configuration.RedirectConfiguration
+
     /// Execute `GET` request using specified URL.
     ///
     /// - parameters:
     ///     - url: Remote URL.
     ///     - deadline: Point in time by which the request must complete.
     public func get(url: String, deadline: NIODeadline? = nil) -> EventLoopFuture<Response> {
-        return self.get(url: url, deadline: deadline, logger: HTTPClient.loggingDisabled)
+        return self.get(url: url, deadline: deadline, logger: HTTPClient.loggingDisabled, redirectConfiguration: nil)
+    }
+
+    /// Execute `GET` request using specified URL.
+    ///
+    /// - parameters:
+    ///     - url: Remote URL.
+    ///     - deadline: Point in time by which the request must complete.
+    ///     - redirectConfiguration: The configuration of redirect handling for this request.
```

Comment:
Can we update these docstrings to explain what `nil` means? This applies to all new methods.

Reply:
Sure. Can we also change the argument name from `redirectConfiguration` to `redirects`? I think there was an earlier suggestion that this is more pithy and I like the idea.

Reply:
No objection from me.